docs/shared/getting-started/start-with-existing-project.md

```diff
@@ -12,12 +12,67 @@ Thanks to [Nx's modular architecture](/getting-started/intro), you can start wit
 Getting started is remarkably simple. You can add Nx to any existing project with a single command:
 
 ```shell
-nx init
+nx@latest init
 ```
 
-_(Note, make sure you have [Nx installed globally](/getting-started/installation) or use `npx` if you're in a JavaScript environment)_
+> _Note: make sure you have [Nx installed globally](/getting-started/installation) or use `npx` if you're in a JavaScript environment_
 
-This command automatically detects your underlying workspace structure, whether it's a monorepo, a single project, or something in between. Nx will then ask you a series of targeted questions about your setup and preferences, using your answers to auto-configure itself seamlessly into your existing workspace. This intelligent setup process ensures you get the benefits of Nx with minimal configuration changes or disruption to your current development workflow.
+This command automatically detects your underlying workspace structure, whether it's a monorepo, a single project, or something in between. Nx will then ask you a series of targeted questions about your setup and preferences, using your answers to auto-configure itself seamlessly into your existing workspace. This intelligent setup process ensures you get the benefits of Nx with minimal configuration changes or disruption to your current development workflow.Add
+
+## Next Steps
+
+Explore your workspace:
+
+- Run `nx graph` to view an interactive graph
+- Run `nx show projects` to see a list of all projects
+- Run `nx show project <project-name>` to view a interactive project detailed view
+- Run `nx <target> <project>` to run a task with Nx, e.g. `nx build my-project`
```

Comment:
goal was to walk through basic discovery of what project exist and the targets on those projects. but can also shorten it to just `nx <target> <project>` if we think it's too verbose